package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vite-plugin-virtual-mpa",
-  "version": "1.10.1",
+  "version": "1.10.2",
```

Comment:
This is patch version not minor version. We have added some new features, should change version to 1.11.0